tests/parse.spec.js

```diff
@@ -232,4 +232,54 @@ describe('Parse', function() {
     ]);
   });
 
+  it('should handle unknown escape sequences inside a value', function() {
+    m.chai.expect(parse([
+      'device: /dev/disk1',
+      'description: "\\]=-01`234567890=-098e"'
+    ].join('\n'))).to.deep.equal([
+      {
+        device: '/dev/disk1',
+        description: ']=-01`234567890=-098e'
+      }
+    ]);
+  });
+
+  it('should delete known escape sequences', function() {
```

Comment:
> Perhaps there should be an optional parameter that a user could supply to drivelist, specifying whether they want these escape sequences preserved or not?

I think it makes a lot of sense. I made the escape sequences go through without issues, and we can handle from the Etcher side if really necessary.